packages/odyssey-storybook/src/components/odyssey-labs/Switch/Switch.stories.tsx

```diff
@@ -89,11 +89,11 @@ export const Default: StoryObj<typeof Switch> = {
       const canvas = within(canvasElement);
       const switchCheckbox = canvas.getByRole("checkbox") as HTMLInputElement;
       if (switchCheckbox) {
-        userEvent.click(switchCheckbox);
+        await userEvent.click(switchCheckbox);
       }
-      expect(switchCheckbox).toBeChecked();
-      axeRun("Switch Default");
-      userEvent.tab();
+      await expect(switchCheckbox).toBeChecked();
+      await axeRun("Switch Default");
+      await userEvent.tab();
```

Comment:
😮

Reply:
Yea, not sure if things changed, but we need to await these async calls or else we get unexpected results